operations/http.go

```diff
@@ -595,6 +595,9 @@ func (ac *legacyClient) PutPatch(incomingPatch patchSubmission) (*patch.Patch, e
 	if resp.StatusCode != http.StatusCreated {
 		return nil, NewAPIError(resp)
 	}
+	if resp.StatusCode == http.StatusBadRequest {
```

Comment:
Could this be an `||` with the if statement on line 595?

Reply:
sorry this is actually unneeded i forgot to remove